dev/preview/previewctl/main.go

```diff
@@ -18,6 +20,15 @@ func main() {
 		TimestampFormat: "2006-01-02 15:04:05",
 	})
 
+	if os.Getenv("GOOGLE_APPLICATION_CREDENTIALS") == "" {
+		if credFile := os.Getenv("PREVIEW_ENV_DEV_SA_KEY_PATH"); credFile != "" {
```

Comment:
What dependency(cies) do we still have on `PREVIEW_ENV_DEV_SA_KEY_PATH`?

Reply:
I see it mentioned in:
1. the dev/image
2. deploy-gitpod action
3. scripts in dev/preview

Is this what you meant by:
> transitioning to VM service accounts that generate short-term tokens

I ask because I was not expecting to see. Perhaps it is just a "bad name"?

Reply:
Nevermind, you answered in `configure-workspace.sh`:

> The name PREVIEW_ENV_DEV_SA_KEY_PATH is borrowed here, and what is actually stored is not the key of the service account, but a configuration file, which is not confidential and does not contain a token.